src/exercise/client/src/Components/Task/Task.js

```diff
@@ -0,0 +1,87 @@
+import "./Task.css";
+import { Checkbox, Button } from "monday-ui-react-core";
+import { Delete } from "monday-ui-react-core/dist/allIcons";
+import { useState } from "react";
+import PropTypes from "prop-types";
+
+function taskDate(dateString) {
+  let date = new Date(dateString);
+  return `${date.getDate()}/${date.getMonth() + 1}/${date.getFullYear()}`;
+}
+
+function taskTime(dateString) {
+  let date = new Date(dateString);
+  let minutes = String(date.getMinutes());
+  if (minutes.length === 1) {
+    minutes = `0${minutes}`;
+  }
+  return `${date.getHours()}:${minutes}`;
+}
+
+export default function Task({
+  taskId,
+  task,
+  date,
+  status,
+  setTasks,
+  deleteTask,
+  changeTaskStatus,
+}) {
+  const [showTask, setShowTask] = useState("taskSection");
```

Comment:
what is it? why its value is string? what does it represent?

Reply:
if its the class name set it properly
we usually will have bollean in the state for such things and in th classname will do something like
className={isTask? "bla":"blabla"}